gdeep/trainer/trainer.py

```diff
@@ -533,11 +709,9 @@ def _init_profiler(
     ) -> None:
         """initialise the profler for profiling"""
         # profiling
-        active: int = 10
-        if not cross_validation:
-            active = n_epochs - 2
-        else:
-            active = k_folds * (n_epochs - 2)
+        active: int = 2
+        if  cross_validation:
```

Comment:
an extra space here.

Reply:
Thank you. It will be improved in the next push.